src/DependencyInjection/Configuration.php

```diff
@@ -17,6 +17,8 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        return new TreeBuilder();
+        $treeBuilder = new TreeBuilder();
+        $treeBuilder->root('simplemde');
```

Comment:
Two questions. What's the purpose of this as it relates to sf 4? Also if it needs to be added I think it should be 'ns_simplemde'